apps/core/lib/core/models/user.ex

```diff
@@ -46,9 +46,22 @@ defmodule Core.User do
     |> validate_format(:email, @valid_email_format)
     |> unique_constraint(:email)
     |> validate_length(:password, @valid_password_length)
+    |> validate_confirmation(:password, message: "does not match password!")
     |> encrypt_passwd()
   end
 
+  def update_changeset(model, params) do
```

Comment:
If you don't want to manage the password in the `registration_changeset` then just extract it to its own changeset.